src/i18n/en.js

```diff
@@ -32,7 +32,7 @@ const urlSiteContact = `${urlSite}/en/contact/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 const urlCreateSubscriberList = `${urlDopplerLegacy}/Lists/List`;
-const urlDraft = `${urlDopplerLegacy}/Campaigns/Draft/`;
+const urlDraft = `${urlDopplerLegacy}`;
```

Comment:
How about making this change also? to avoid confusion
```suggestion
const homeUrl= `${urlDopplerLegacy}`;
```
You could make it in a separate commit.